src/config/ServerConfig.ts

```diff
@@ -80,3 +81,17 @@ export const setServerConfig = (svcConfig: ServerConfig) => {
 
   computeValidDurations(serverConfig);
 };
+
+export const isIstioNamespace = (namespace: string): boolean => {
+  if (namespace === serverConfig.istioNamespace) {
+    return true;
+  }
+  if (serverConfig.istioComponentNamespaces) {
+    for (const ns of Object.values(serverConfig.istioComponentNamespaces)) {
```

Comment:
The following is equivalent  `Object.values(serverConfig.istioComponentNamespaces).includes(namespace)`.

Reply:
Anytime we can replace iteration with declaration is a good thing.

Reply:
You kids just love your condensed code...  OK, I'll change it.